.github/workflows/reset-azure-environment.yml

```diff
@@ -0,0 +1,193 @@
+          when you open the slot, before the app name in parenthesis.
+      source-slot-name:
+        required: false
+        type: string
+        default: Production
+        description: >
+          The slot name of the web app you want to reset the destination web app with. What you see at the top of the
+          blade on the Azure Portal, when you open the slot, before the app name in parenthesis.
+      resource-group-name:
+        required: true
+        type: string
+        description: Name of the resource group.
+      database-connection-string-name:
+        required: true
+        type: string
+        description: >
+          The name of the database connection string. The Connection String Name property specifies the name of the
+          named connection string that the SQL Workflow Instance Store should use to connect to the persistence
+          database.
+      master-database-connection-string-name:
+        required: true
+        type: string
+        description: >
+          The name of the master database connection string. The master database is the primary configuration database
+          in SQL Server. It contains information on all the databases that exist on the server. The Connection String
+          Name property specifies the name of the named connection string that the SQL Workflow Instance Store should
+          use to connect to the persistence database.
+      storage-connection-string-name:
+        required: true
+        type: string
+        description: >
+          The name of the storage connection string. The Connection String Name property specifies the name of the
+          named connection string that the SQL Workflow Instance Store should use to connect to the persistence
+          database.
+      service-objective-name:
+        required: false
+        type: string
+        default: S0
+        description: The name of the service objective to scale the database to, e.g. B, S1, S4, P4, P6.
+      blob-container-exclude-list:
+        required: false
+        type: string
+        default: "@()"
+        description: >
+          PowerShell string array with the name of the excluded Blob Container(s), e.g.
+          "@(\"not-media\")" or "@(\"not-media\", \"bad-container\")". The parameter must be a PowerShell string array.
+          When valid values are provided for "blob-container-include-list", then "blob-container-exclude-list" is not
+          taken into consideration.
+      blob-container-include-list:
+        required: false
+        type: string
+        default: "@(\"media\")"
+        description: >
+          PowerShell string array with the name of the included Blob Container(s), e.g.
+          "@(\"media\")" or "@(\"media\", \"cool-container\")". The parameter must be a PowerShell string array. When
+          valid values are provided, it cancels out "blob-container-exclude-list".
```

Comment:
It's unclear what "cancels out" means. If the point is that if `blob-container-include-list` is valid then exclude-list won't be used, then repeated the description under the latter. Also explain what "valid" is, because the defaults seem valid too.

Same for folder.